src/App.vue

```diff
@@ -1612,6 +1629,14 @@ export default {
 			}
 		})
 	},
+	beforeUnmount() {
+		if (this.mq_system_dark !== null) {
+			this.mq_system_dark.removeEventListener(
+				'change',
+				this.updateSystemDark,
+			)
+		}
+	},
```

Comment:
this is a bit useless as the App will only be unmounted when the page is destroyed